README.md

```diff
@@ -57,7 +57,7 @@ If there are no errors, it means that:
 The `data` that the gem consumes can be anything that responds to at least the following (all these should return strings except the giftaid, which is a boolean):
 
 * `data.amount`: The amount of money to be donated, in currency units. This can be an amount with cents, and will be converted to cents by the gem. For example: 10.50
-* `data.currency`: The currency that the amount is in, should be a valid ISO 4217 code for currencies and supported by the Money gem that this gem uses to handle money. [Check here for a list of supported currencies in JSON format]((https://github.com/RubyMoney/money/tree/e2773f7859b268965fa003e2630ed58e7e96ac58/config)
+* `data.currency`: The currency that the amount is in, should be a valid ISO 4217 code for currencies and supported by the Money gem that this gem uses to handle money. [Check here for a list of supported currencies in JSON format](https://github.com/RubyMoney/money/tree/e2773f7859b268965fa003e2630ed58e7e96ac58/config)
```

Comment:
Reworded:
The currency that the amount is in, should be a valid ISO 4217 code for currencies and supported by the Money gem, which is used by this gem to handle money.